### PR TITLE
usage example for sprite set velocity

### PR DESCRIPTION
**Pull request by kay-kaushik**
# Description
This pull request features usage exaples for sprite_set_velocity(). The usage example is clear and concise to help the user understand how to implement the function.

# Features
+ clear and concise code
+ gifs
+ perfect example

# Changes
5 files have been added. 2 code, 1 text, 1 zip folder and a gif file.

# Benefits
These high quality usage examples give more context to the functions. This will help users reading the documentation to better understand the function. This will result in faster development.

# Testing
All the changes have been tested on Kartik Kaushik's system

# Testing Images

Screenshot 2024-09-05 195900

Screenshot 2024-09-05 195851

### CODE REVIEW

File: src/assets/usage-examples-code/sprites/sprite_set_velocity/sprite_set_velocity-6-sprite-velocity.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+### sprite_set_velocity() usage example
```
Comment (omckeon) [inline src/assets/usage-examples-code/sprites/sprite_set_velocity/sprite_set_velocity-6-sprite-velocity.txt:1]: Improve this heading

File: src/assets/usage-examples-code/sprites/sprite_set_velocity/sprite_set_velocity-6-sprite-velocity.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
+
+The following code is an example of using [sprite set velocity](https://splashkit.io/api/sprites/#sprite-set-velocity) which sets the velocity of a sprite on a splashkit window.
```
Comment (omckeon) [inline src/assets/usage-examples-code/sprites/sprite_set_velocity/sprite_set_velocity-6-sprite-velocity.txt:3]: Update this to be a bit clearer and use capital first letters on the function name.
eg. "The following code is an example of using [Sprite Set Velocity](https://splashkit.io/api/sprites/#sprite-set-velocity) which sets the velocity (speed and direction) of a sprite on a window.

File: public/usage-examples-images-gifs/sprites/sprite_set_velocity-6-sprite-velocity.gif
Comment (omckeon) [inline public/usage-examples-images-gifs/sprites/sprite_set_velocity-6-sprite-velocity.gif:1]: Can you simplify the name of the image? And then make sure to update this PR to use the instructions for including resources.

**Review by omckeon — CHANGES_REQUESTED** [review]
Just need to update a few things mentioned in the comments

**Comment by omckeon** [comment]
Make sure you have run `npm run build` successfully as well. (The check should not be failing.)

File: src/assets/usage-examples-code/sprites/sprite_set_velocity/sprite_set_velocity-6-sprite-velocity.txt (hunk 1 above)
Comment (kay-kaushik) [reply src/assets/usage-examples-code/sprites/sprite_set_velocity/sprite_set_velocity-6-sprite-velocity.txt]: I have improved the header

File: src/assets/usage-examples-code/sprites/sprite_set_velocity/sprite_set_velocity-6-sprite-velocity.txt (hunk 2 above)
Comment (kay-kaushik) [reply src/assets/usage-examples-code/sprites/sprite_set_velocity/sprite_set_velocity-6-sprite-velocity.txt]: I have updated the description

File: public/usage-examples-images-gifs/sprites/sprite_set_velocity-6-sprite-velocity.gif
Comment (kay-kaushik) [reply public/usage-examples-images-gifs/sprites/sprite_set_velocity-6-sprite-velocity.gif:1]: I have updated the name of the image file to player-run.png and I also have added it to the resource folder

**Review by NidhishaPahade — COMMENTED** [review]
Everything looks good to me.

**Pull request merged**